committee-steering/governance/sig-governance.md

```diff
@@ -117,13 +116,21 @@ curation from other SIG participants
 
 ### Tech Lead
 
-- *Optional Role*: SIG Technical Leads
-  - Establish new subprojects
-  - Decommission existing subprojects
-  - Resolve X-Subproject technical issues and decisions
-  - Number: 2-3
-  - Membership tracked in [sigs.yaml]
-  - Role description in [technical-lead.md]
+- Number: 2+
+- Membership tracked in [sigs.yaml]
+- *MUST* Approve & facilitate the creation of new subprojects
+- *MUST* Approve & facilitate decommissioning of existing subprojects
+- *MUST* Resolve cross-Subproject and cross-SIG technical issues and decisions
+  or delegate to another Lead as needed
+- *MUST* in conjunction with the Chairs identify, track, and maintain the
+  metadata of the SIGs enhancement proposals for the current release and serve
+  as point of contact for the release team, but *MAY* delegate to other
+  contributors to fulfill these responsibilities
+- *MUST* Review & Approve SIG Enhancement Proposals, but *MAY* delegate to
+  other contributors to fulfill these responsibilities for individual proposals
+
+Additional information on the Tech Lead role can be found in
+[technical-lead.md]; within the [Chair & TL Contributor Documentation].
```

Comment:
That document still describes the Tech Lead role as optional

Reply:
that doc is a general 'what sort of stuff does a tl do' and not part of governance docs, it can be updated anytime out of band 👍